<?xml version="1.0" encoding="UTF-8"?>
<launch>

	<node pkg="traj_gen" type="traj_gen_client" name="traj_gen_ui" output="screen">
		<param name="world_frame_id" value="/world" />
		<param name="waypoint_topic" value="/waypoint" />
		<param name = "spline_save_dir" value = "$(find traj_gen)/spline_log/spline_log.txt"/>
	</node>
	<node name="rviz" pkg="rviz" type="rviz" args="-d $(find traj_gen)/rviz/rviz.rviz"/>

</launch>


